tests/ui/components/questions/questionsubmissioncomponent.js

```diff
@@ -0,0 +1,36 @@
+import { mount } from 'enzyme';
+import mockManager from '../../../setup/managermocker';
+import DOM from '../../../../src/ui/dom/dom';
+import QuestionSubmissionComponent from '../../../../src/ui/components/questions/questionsubmissioncomponent';
+import StorageKeys from '../../../../src/core/storage/storagekeys';
+import SearchStates from '../../../../src/core/storage/searchstates';
+
+DOM.setup(document, new DOMParser());
+let COMPONENT_MANAGER, defaultConfig;
+beforeEach(() => {
+  COMPONENT_MANAGER = mockManager();
+  const bodyEl = DOM.query('body');
+  DOM.empty(bodyEl);
+  DOM.append(bodyEl, DOM.createEl('div', { id: 'test-component' }));
+
+  defaultConfig = {
+    container: '#test-component',
+    entityId: 1234
+  };
+});
+
+describe('QASubmission component', () => {
+  it('listens to updates to VERTICAL_RESULTS in global storage', () => {
```

Comment:
This is testing that the component appears only when there is at least one result, correct?

Reply:
oh, results count doesn't matter here! I'll remove results and reusltscount from the data to make that clear